src/absil/ilreflect.fs

```diff
@@ -685,10 +707,17 @@ let queryableTypeGetMethod cenv emEnv parentT (mref:ILMethodRef) =
         let cconv = (if stat then BindingFlags.Static else BindingFlags.Instance)
         let methInfo = 
             try 
+#if FX_RESHAPED_REFLECTION
+              parentT.GetMethod(mref.Name,cconv ||| BindingFlags.Public ||| BindingFlags.NonPublic,
+                                null,
+                                argTs,
+                                (null:obj[]))
+#else
               parentT.GetMethod(mref.Name,cconv ||| BindingFlags.Public ||| BindingFlags.NonPublic,
                                 null,
                                 argTs,
-                                (null:ParameterModifier[])) 
+                                (null:ParameterModifier[]))
+#endif
```

Comment:
The change is only the final parameter it looks like. Why not #ifdef just the last line?